src/trialscope.ts

```diff
@@ -233,7 +250,9 @@ export class TrialScopeQuery {
    * @return {string} the TrialScope GraphQL query
    */
   toQuery(): string {
-    let baseMatches = `conditions:[${Array.from(this.getTrialScopeConditions()).join(', ')}], baseFilters: { zipCode: "${this.zipCode}"`;
+    let baseMatches = `conditions:[${Array.from(this.getTrialScopeConditions()).join(
+      ', '
+    )}], baseFilters: { zipCode: "${this.zipCode}"`;
```

Comment:
This is a funky one. The line itself is a mess but I'm not sure this makes it any better.

Reply:
I've taken a stab at improving the line to make it more readable.